src/include/ci/efhw/common_ci2linux.h

```diff
@@ -11,7 +11,7 @@
 #endif
 #include <stdbool.h>
 
-#if ! (defined bool) && ! (defined __cplusplus)
+#if ! (defined bool) && ! (defined __cplusplus) && (__STDC_VERSION__ < 202311L)
```

Comment:
Could we change this to be an inclusive check of the final version that supports stdbool instead (i.e., C17 with `201710L`)? Testing with gcc-13.2.0 which admittedly only partially supports C23 (using `--std=c2x`) this check is bypassed because (presumably) a placeholder value was used for `__STDC_VERSION__`:
```
$ cat test.c
#include <stdio.h>

int main() {
  printf("__STDC_VERSION__ = %ld\n", __STDC_VERSION__);
  return 0;
}

$ gcc test.c -o test --std=c2x
$ ./test
__STDC_VERSION__ = 202000
```

Updating the clause as below worked for me:
```suggestion
#if ! (defined bool) && ! (defined __cplusplus) && (__STDC_VERSION__ <= 201710L)
```

Would it also be possible to add a brief comment explaining that C23 deprecated this header (and give context that the magic number represents C17) please?

Reply:
Sure, done.